src/scenes/main-scene/index.ts

```diff
@@ -4,7 +4,9 @@ export class MainScene extends Phaser.Scene {
   fireButton: Phaser.Input.Keyboard.Key;
   cursors: Phaser.Types.Input.Keyboard.CursorKeys;
   player: Phaser.Types.Physics.Arcade.ImageWithDynamicBody;
-  coin: Phaser.Types.Physics.Arcade.ImageWithDynamicBody;
+  coin: Phaser.GameObjects.Image;
```

Comment:
We're not using `coin` anymore, maybe best to remove it so it doesn't get confused with the `coin` from method params

Reply:
good point, will do